conf/app.routes

```diff
@@ -60,6 +60,7 @@ GET           /journey-cache/:journey/values/:key
 POST          /journey-cache/:journey                                                                                 @uk.gov.hmrc.tai.controllers.JourneyCacheController.cached(journey: String)
 DELETE        /journey-cache/:journey                                                                                 @uk.gov.hmrc.tai.controllers.JourneyCacheController.flush(journey: String)
 DELETE        /journey-cache/:journey/:empId                                                                          @uk.gov.hmrc.tai.controllers.JourneyCacheController.flushWithEmpId(journey: String, empId: Int)
+DELETE        /test-only/journey-cache/delete-update-income                                                                     @uk.gov.hmrc.tai.controllers.JourneyCacheController.delete()
```

Comment:
My understanding is anything in this file will expose an endpoint. 
This should be placed in the testOnlyDoNotUseInAppConf.routes. Right? 
Example: https://github.com/hmrc/plastic-packaging-tax-registration-frontend/blob/main/conf/testOnlyDoNotUseInAppConf.routes

Reply:
Yes. It's already there in `testOnlyDoNotUseInAppConf`. Will remove it from here.